lib/utils.js

```diff
@@ -47,11 +39,28 @@ const defaults = {
   messageOnly: false,
   tz: 'UTC',
   newline: false,
-  sync: false
+  sync: false,
+  pinoLevelToSyslogSeverity
+}
+
+function buildOptions (options = {}, args = {}) {
+  const opts = Object.assign({}, defaults, options, args)
+
+  if (options.customLevels == null) {
+    return opts
+  }
+
+  for (const [key, { level, syslogSeverity }] of Object.entries(options.customLevels)) {
+    pinoLevels[key] = level
+    pinoLevelToSyslogSeverity.set(pinoLevels[key], severity[syslogSeverity])
+  }
```

Comment:
This should verify that the destination severity is a known severity. I am not clear what should be done if a severity is not recognized.

Reply:
Maybe we could choose severity.error by default and if there is a misconfiguration the syslog administrator will find out about that problem?

Reply:
This would need to be clearly documented.